cmd/run.go

```diff
@@ -295,8 +295,21 @@ func runSynchronizer(cfg config.Config, etherman *etherman.Client, ethTxManager
 	}
 	zkEVMClient := client.NewClient(trustedSequencerURL)
 
+	//var etherman2 *etherman.Client = nil
+
+	etherManForL1 := []synchronizer.EthermanInterface{}
+	// If synchronizer are using sequential mode, we only need one etherman client
+	if cfg.Synchronizer.UseParallelModeForL1Synchronization {
+		for i := 0; i < int(cfg.Synchronizer.L1ParallelSynchronization.NumberOfParallelOfEthereumClients); i++ {
+			eth, err := newEtherman(cfg)
+			if err != nil {
+				log.Fatal(err)
+			}
+			etherManForL1 = append(etherManForL1, eth)
+		}
+	}
```

Comment:
synchronizer still need a isntance of etherman because is used in some places like: 
* `func Sync`: VerifyGenBlockNumber, HeaderByNumber
* To get forkID info...
To avoid any conflict I think that is better to create a new instances